adafruit_ble/advertising/__init__.py

```diff
@@ -248,8 +243,8 @@ def from_entry(cls, entry):
 
     @property
     def rssi(self):
-        """Signal strength of the scanned advertisement. Only available on Advertisement's created
-           from ScanEntrys. (read-only)"""
+        """Signal strength of the scanned advertisement. Only available on Advertisements created
+           from ScanEntry objects. (read-only)"""
```

Comment:
I wouldn't reference ScanEntry since it is internal to this API. I'd say "Only available on Advertisements returned from \`start_scan\`."

Reply:
`ScanEntry` is mentioned in the docstrings for `from_entry()` and `matches()`, but I agree this is likely to be used by the end user, and the others are not.